### ISSUE TITLE
ndep is not using the right years

### ISSUE THREAD

**Issue by mnlevy1981** [mnlevy1981/frontera_highres#9]
Want `ndep_shr_stream_year_first = 1957` so that year 0001 is 1958 ndep

**Comment by mnlevy1981** [comment]
Alternatively, do we want to set `RUN_STARTDATE=`1958-01-01`? would require updating `datm_in` to make sure everything is aligned.

Pro: easier to compare to obs
Con: harder to compare to other FOSI runs

**Comment by mnlevy1981** [comment]
If we change `RUN_STARTDATE`, want

```
  streams = "datm.streams.txt.CORE_IAF_JRA.GCGCS.PREC 1958 1958 2021",
      "datm.streams.txt.CORE_IAF_JRA.GISS.LWDN 1958 1958 2021",
      "datm.streams.txt.CORE_IAF_JRA.GISS.SWDN 1958 1958 2021",
      "datm.streams.txt.CORE_IAF_JRA.NCEP.Q_10 1958 1958 2021",
      "datm.streams.txt.CORE_IAF_JRA.NCEP.SLP_ 1958 1958 2021",
      "datm.streams.txt.CORE_IAF_JRA.NCEP.T_10 1958 1958 2021",
      "datm.streams.txt.CORE_IAF_JRA.NCEP.U_10 1958 1958 2021",
      "datm.streams.txt.CORE_IAF_JRA.NCEP.V_10 1958 1958 2021"
```

and

```
streams = "drof.streams.txt.rof.iaf_jra 1958 1958 2021"
```

**Comment by mnlevy1981** [comment]
fixed in a0efb46
